src/js/components/products/detail.tsx

```diff
@@ -0,0 +1,78 @@
+import BreadCrumb from '@salesforce/design-system-react/components/breadcrumb';
+import PageHeader from '@salesforce/design-system-react/components/page-header';
+import i18n from 'i18next';
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+import { Trans } from 'react-i18next';
+import { connect } from 'react-redux';
+import { Link, RouteComponentProps } from 'react-router-dom';
+
+import { AppState } from '@/store';
+import { Product } from '@/store/products/reducer';
+import {
+  selectProduct,
+  selectProductSlug,
+  // selectProductNotFound,
+} from '@/store/products/selectors';
+import routes from '@/utils/routes';
+
+type Props = {
+  product?: Product | null;
+  productSlug?: string;
+} & RouteComponentProps;
+
+const ProductDetail = ({ product, productSlug }: Props) => (
+  <DocumentTitle title={`${i18n.t(productSlug)} | ${i18n.t('MetaShare')}`}>
```

Comment:
No need to run translation on values that come from the database -- if they need to be translated, that'll happen on the DB side. So we only wrap hard-coded front-end strings in translation flags.